schema.graphql

```diff
@@ -404,3 +405,14 @@ type CLPool_Swap {
   timestamp: Timestamp! @index
   chainId: Int! @index
 }
+
+scalar Decimal
+
+type TokenPrice {
+  id: ID! # Unique identifier for the record, could be a combination of token address and chain ID
```

Comment:
chain id , token address and block number it seems. Maybe block timestamp is nicer than block number in future?